driver-scala/src/main/scala/org/mongodb/scala/MongoCollection.scala

```diff
@@ -151,19 +151,19 @@ case class MongoCollection[TResult](private val wrapped: JMongoCollection[TResul
    * Counts the number of documents in the collection.
    *
    * '''Note:'''
-   * For a fast count of the total documents in a collection see [[estimatedDocumentCount()]]
+   * For a fast count of the total documents in a collection see [[estimatedDocumentCount()*]]
```

Comment:
What does the `*` signify?

Reply:
Not 100% sure but it solves the ambiguity warning that you saw previously.